backend/src/main/java/com/soen341/instagram/exception/comment/UnauthorizedRightsException.java

```diff
@@ -0,0 +1,13 @@
+package com.soen341.instagram.exception.comment;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
```

Comment:
Can you add a comment explaining the cases why we would want to throw this error?

Reply:
Based off the `CmmentService.java` file, it seems the error is meant to be thrown in the case an a user tries to delete a comment on a post that isn't theirs. Still, it would be nice to add a little explanation at the beginning of the file so that anyone looking at our code for the first time can easily understand the intended purpose of this class.

Reply:
Comment added